work_items/schema_directory.md

```diff
@@ -0,0 +1,30 @@
+# Schema Directory
+
+The DIF Schema Directory will act as a source of curated references to aid 
+developers in finding schemas that address their needs. When seeking to add 
+to the directory, the group will first seek out existing schemas from external 
+sources that cover the motivating need for the addition. If no schema exists 
+externally that addresses the need, the group may elect to generate and publish 
+new schemas and publish them within the directory project/repo.
+
+## Work Item Owners
+- Daniel Buchner (@github, email@example.com, or both)
+- Martin Riedel (@rado0x54)
+
+## Outcome
+This will be a long-lived work item, but an indication of success will be that 
+we have assembled a directory with enough coverage across a wide enough array 
+of use cases and verticals that it becomes well-known in the wider community 
+as valuable place to look for schemas when questions arise about which to use 
+in a given circumstance.
+
+## Deliverables
+- Process for evaluation and inclusion of a new schema category or reference
+- Initial directory populated with a significant enough number of schemas to provide value
+- Publication of the directory on the DIF website
+
+## Meetings
+- Meetings will be held once a month, with async eval of PRs for additions being 
+  the primary mode of collaboration.
+- Meetings will primarily be focused on review of any outstanding issues.
```

Comment:
Add prior art section in relation to VC Working Group 2.0, specifically around credentialSchema.

Reply:
https://w3c-ccg.github.io/vc-json-schemas/v2/index.html ?